app/controllers/mixins/generic_button_mixin.rb

```diff
@@ -89,5 +89,14 @@ def button
         render_flash unless performed?
       end
     end
+
+    def evaluate_button(action, allowed_actions)
+      unless allowed_actions.key?(action)
+        raise ActionController::RoutingError, _('invalid button action')
```

Comment:
Probably a capital `I` in `invalid` and it could be a one-liner.

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller/explorer.rb#L75

https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller/explorer.rb#L79

https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller/explorer.rb#L185
 they all have `invalid button action` with `i`